assets/css/_info-bubble.scss

```diff
@@ -181,18 +181,16 @@
 
   .non-variant {
     float: right;
-    margin-right: 8px;
-    margin-top: 5px;
-    margin-bottom: 8px;
-    width: 150px;
-    height: $info-bubble-button-size;
-    line-height: $info-bubble-button-size;
-
-    user-select: none;
-
-    white-space: nowrap;
-
     text-align: right;
+    margin-top: 9px;
```

Comment:
Not sure if this is what's causing this now, but in my screenshot the buttons are now mis-aligned by one pixel _below_ the ones on the left:

<img width="307" alt="screenshot 2016-10-03 21 12 57" src="https://cloud.githubusercontent.com/assets/2553268/19059642/6e359154-89af-11e6-9812-fa659299c93c.png">

(Tested on Chrome and Firefox, on a Macbook with a Retina monitor.)